README.md

```diff
@@ -97,6 +97,20 @@ We use Code generation to generate our types for our GraphQL queries and mutatio
 - Each query and mutation should have a unique name.
 - Since query analysis for type generation occurs statically we cant place dynamic variables with in query strings we instead have to hard code the variable in the query or pass it in as query variable.
 
+## How to get data for your feature
+If you need more data to be able to test out your feature locally the easiest way to do it is to populate the local db using real data from the staging or production environments.
+- First step is to locate where you want to fetch your data from if it exists in either the staging or production environments.
+- After identifying where to get your data from you should try to ssh into one of the db servers used for staging or prod. The urls for these dbs can be located in the `fabfile.py` located in the evergreen directory.
```

Comment:
i think this list would benefit from being a numbered list of terse instructions. 
1. SSH into the production server - ssh url can be in the fabfile located here ('here' should be a link)
2. Ensure you are connected to a secondary node before proceeding
3. Run `mongo` command to open the mongo shell
4. Query your data, e.g. - show example of a mongo query maybe ?

...